### ISSUE TITLE
Terminator display freeze

### ISSUE THREAD

**Issue by 0011001011** [gnome-terminator/terminator#411]
`terminator -g /dev/null` 
> still happens

**Describe the bug**
During normal usage of terminator, the display completely freeze (even the cursor actually). It usually happens when using ALT+TAB or other CMD stuff, but not every time.

The terminal still works and accept inputs so you can sort of continue to use it in blind mode.

The only thing to do is either : 

- wait, usually comes back after seconds (>5s)
- switch terminal (always refresh the display, but often freezes on the very next frame)
- click with the mouse in th terminator window (same comment)

Using a htop or something constantly refreshing the terminal does not solve it.

**To Reproduce**
Every launch on my machine, which is  a fresh ubuntu gorilla install. Tried with terminator from repo, and from the latest release on the mattrose/terminator repo.

**Expected behavior**
No freeze

**Screenshots**
NA

**Desktop (please complete the following information):**
 - Linux  5.8.0-45-generic 51-Ubuntu SMP x86_64 x86_64 x86_64 GNU/Linux
 - Ubuntu gorilla
 - terminator 2.0.1

**Additional context**
May be related to this 

https://bbs.archlinux.org/viewtopic.php?id=259770

**Comment by mattrose** [comment]
This is https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/3369 and it affects far more than terminator unfortunately.  It was originally brought up here in #284. The gnome-shell bug seems to be fixed now, but I don't know if Ubuntu has picked up those changes.

Personally, when I went to ubuntu 20.10 I installed https://github.com/justinrdonnelly/wandering-pixel and that fixed the freezing problem.
